recipes/discasm/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.2" %}
+
+package:
+    name: discasm
+    version: {{ version }}
+
+source:
+  fn: DISCASM-v{{ version }}.tar.gz # [unix]
+  url: https://github.com/DISCASM/DISCASM/releases/download/DISCASM-v{{ version }}/DISCASM-v{{ version }}.tar.gz # [unix]
+  md5: 571b2265018608298752f40f3f375b0f # [unix]
+
+build:
+    number: 0
+    skip: true  # [py3k]
+    skip: True  # [osx]
+    # The reason for skipping osx is that trinity >=2.4 is not available in the osx channel
+
+requirements:
+    build:
+    run:
+        - python
+        - perl
+        - star >=2.4
+        - trinity >=2.4
+        - oases >=0.2
+
+test:
+    commands:
+        - DISCASM --help > /dev/null 2>&1
+        # DISCASM requires the environment variable TRINITY_HOME to be specified,
+        # but can find it if it is in the PATH variable.
```

Comment:
Can you explain this comment? It not super clear.

Reply:
Sorry. I had written the first part of the comment (the code simply checked for TRINITY_HOME and terminated if it was not defined). But that made it impossible to pass the installation test. I contacted Brian Haas, and then wrote some code to find the TRINITY_HOME directory and set the variable. Then did a pull request and Brian added that code into the distribution. So rather than removing the first comment, I added to it. The code is setting TRINITY_HOME for the other software that is invoked by DISCASM (i.e. trinity).

Reply:
I will try to make the comment more clear.